src/App/Document.cpp

```diff
@@ -856,6 +857,12 @@ Document::Document(const char* documentName)
                       0,
                       PropertyType(Prop_Hidden | Prop_ReadOnly),
                       "Link of the tip object of the document");
+
+    if (!myName.empty())
+        myEditTime = fs::last_write_time(fs::absolute(fs::path(FileName.getValue())));
+    else
+        myEditTime = std::chrono::system_clock::to_time_t(std::chrono::system_clock::now());
+

```

Comment:
This check apparently is not enough, and possibly the cause for the failures. `myName` can be non-empty even for new files (for example `Unnamed`). Adding a check that this path exists also doesn't seem to help.